chrome-ext/styles.css

```diff
@@ -0,0 +1,24 @@
+*{
+    margin: 0;
+    padding: 0;
+    font-family: sans-serif;
+    font-size: 4rem;
+    font-weight: bolder ;
+    letter-spacing: 1rem;
+}
+
+
+main{
+
```

Comment:
It is best practice to remove any unused lines of code for clarity.

Reply:
true... noted thanks!